lib/container.js

```diff
@@ -364,7 +364,7 @@ export default class Container {
     });
   }
 
-  registerDevice(token, type) {
+  registerDevice(token, type, bundleName) {
```

Comment:
Better to change the parameter to `topic` to align with the server naming. And mention the usual value is bundleName.